src/cfi.rs

```diff
@@ -1825,8 +1892,8 @@ impl<'input, Endian> UnwindTableRow<'input, Endian>
     /// >   <tr><td>Vector Mask Registers 0–7</td>        <td>118-125</td> <td>%k0–%k7</td></tr>
     /// >   <tr><td>Reserved</td>                         <td>126-129</td> <td></td></tr>
     /// > </table>
-    pub fn registers(&self) -> &[RegisterRule<'input, Endian>] {
```

Comment:
Not sure what the common use case for this data is, but I suspect there will be cases where the user needs to handle every rule. So maybe we should still provide an API that returns the array. Otherwise I think iterating over the rules will be `O(n^2)`.